Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -30,6 +30,12 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
         /// </summary>
         private const int maximumTouchableVolumeSize = 1000;
 
+        /// <summary>
+        /// The offset that the poke pointer has from the source pose when the index finger pose is not available.
+        /// This value puts the pointer slightly in front of the source pose's origin, oriented according to the source pose's rotation
+        /// </summary>
+        private const float sourcePoseOffset = 0.075f;
```

Comment:
I am not sure if this should be private - could there be a situation where the developer would want to modify the value from code (perhaps based on the type/model of the device)?